### PR TITLE
Suggestions view

### PR DESCRIPTION
**Pull request by moritzbeier**
This adds another table where the search and trending suggestions are previewed.

### CODE REVIEW

File: extension/content/loader.mjs [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 const ONE_DAY = 1000 * 60 * 24;
 const LOCALES_URL =
-  "https://hg.mozilla.org/mozilla-central/raw-file/tip/browser/locales/all-locales";
+  "https://raw.githubusercontent.com/mozilla/gecko-dev/refs/heads/master/browser/locales/all-locales";
```
Comment (moritzbeier) [inline extension/content/loader.mjs:9]: This significantly improved the loading time for me (fetching the `hg.mozilla.org` url takes up to 17 _seconds_) but I'm not sure it's a good idea.

**Review by Standard8 — CHANGES_REQUESTED** [review]
Thank you for working on this. As mentioned in-line, I think we should change improving the startup to a separate patch - there's a few things we could do there.

The rest of the patch looks good, and generally works well. There's one part that I find a bit misleading - in some cases (wikipedia, eBay) they don't have suggestions for "Cute Kitten".

I think either we need to change that to something more general (but then we might also struggle with some of the language specific engines), or maybe it is enough to show that "0 results were found" assuming there were no actual errors in communicating with the server. Another option might be allowing to change the search term, though I'm not sure that's totally necessary.

I'm open to ideas here - the main aim is to clarify if the suggestions are working or not, but displaying the results can be useful as well.

File: extension/content/loader.mjs (hunk 1 above)
Comment (moritzbeier) [reply extension/content/loader.mjs]: We could definitely cache the locales but to you see an issue with pinging Github in particular? I thought mozilla-central will move to Github soon anyway and their infrastructure is just so much faster than ours.

**Review by Standard8 — APPROVED** [review]
Thank you, this looks good. The way the results are handled are much better.

One thought I did have - what do you think about auto-scrolling to the top of the page when the results are received? I think it might be easier/make it clearer that they're there.

**Comment by Standard8** [comment]
Additional thought, please could you also bump the version number in `package.json` and in `extension/manifest.json` to `2.6.0`?

That'll make it easier for us to do the release.

**Comment by moritzbeier** [comment]
This should make it clear

**Pull request merged**